mypy.ini

```diff
@@ -0,0 +1,13 @@
+[mypy]
+  python_version = 2.7
```

Comment:
I feel that an annotated py2 release would be worth it, but I'm definitely in favor of moving to python 3!